server/storage/storage.go

```diff
@@ -135,6 +139,22 @@ func (ps *coreStorage) LoadRegionsOnce(ctx context.Context, f func(region *core.
 	return nil
 }
 
+// LoadRegion loads one region from storage.
+func (ps *coreStorage) LoadRegion(regionID uint64, region *metapb.Region) (ok bool, err error) {
```

Comment:
Seem this function only be used in the tests

Reply:
It is an implementation of interface.

Reply:
Yes, but it's only used for tests.